.github/workflows/main.yml

```
@@ -126,7 +126,7 @@ jobs:
       - name: Make pinetime-mcuboot-app
         run:  |
           cd build
-          make pinetime-mcuboot-app
+          ninja pinetime-mcuboot-app
```

```suggestion
          cmake --build build --target pinetime-mcuboot-app
```

when already touching this part, you could switch away from manually calling the build command (`make` vs `ninja`), and needing to `cd build` beforehand. The two commands could be replaced by using just `cmake --build build --target pinetime-mcuboot-app` called from the source directory

---

That is a good point, we should try and follow modern CMake best practices where possible and make things universal

---

Made the change, thanks!